pkg/cmd/pr/shared/preserve_test.go

```diff
@@ -78,9 +78,8 @@ func Test_PreserveInput(t *testing.T) {
 
 			io, _, _, errOut := iostreams.Test()
 
-			tf, tferr := tmpfile()
+			tf, tferr := tmpfile(t)
```

Comment:
Could we drop the `tmpfile()` function here and inline its contents?
Also, since this is in a loop, could we reuse a `t.TempDir()` initialized outside of the loop?